registrations/forms.py

```diff
@@ -41,6 +42,24 @@ class RegistrationDetailsForm(forms.Form):
         widget=forms.CheckboxSelectMultiple,
     )
 
+    terms_and_conditions = forms.MultipleChoiceField(
+        label=(
+            format_html(
+                "{} <b>{}</b> {}",
+                "Please read the",
+                mark_safe('<a href="terms_and_conditions">Terms & Conditions.</a>'),
```

Comment:
I think we should rather be using the `reverse` function here to get the URL, rather than hardcoding it in

Reply:
Reading the docs, seems like we can have HTML in the first string, and we only need `mark_safe` if we have a variable that contains HTML. So we can do something like:
```
format_html('Please read the <a href="{}">Terms & Conditions</a>.', reverse('....'))
```

Reply:
how does the reverse function work? would I just include the name of the url in the brackets instead of the url?

Reply:
Yup, pretty much. For example: https://github.com/praekeltfoundation/nurseconnect-registration/blob/develop/registrations/views.py#L12

Docs are here: https://docs.djangoproject.com/en/2.1/ref/urlresolvers/#reverse